### PR TITLE
New Extension: TinyFaces NFT Extension

### PR DESCRIPTION
**Pull request by peduarte**
## Description

Explore the TinyFaces NFT Collection. 

Currently this extension supports:
- Search TinyFaces by ID
- Search TinyFaces by Traits
- Display a Detail view including selected ID's data

In the future I'd like to improve this extension with:
- ~Search by `owner` (address or ENS)~ done
- Improve search by `owner` UI. Currently it just returns the list of NFTs, but would be cool to have a dedicated view for this. Needs some thinking, will do in a future PR
- Improve Empty View
- Add a dedicated birthday command

## Screencast

https://user-images.githubusercontent.com/372831/189072733-7bdf84eb-31e3-4b88-8240-fa48cfe9d300.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/tinyfaces-nft/src/tinyfaces.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+  return (
+    <List
+      throttle
+      isShowingDetail
+      isLoading={isLoading}
+      searchBarPlaceholder={SEARCH_PLACEHOLDERS[searchType]}
+      searchText={searchText}
+      onSearchTextChange={setSearchText}
+      onSelectionChange={setSelectedId}
```
Comment (pernielsentikaer) [inline extensions/tinyfaces-nft/src/tinyfaces.tsx:125]:
```suggestion
```
Leftover?

File: extensions/tinyfaces-nft/src/tinyfaces.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    <WagmiConfig client={client}>
+      <Tiny />
+    </WagmiConfig>
+  );
+}
+
+export function Tiny() {
+  const [searchText, setSearchText] = useState<undefined | string>(undefined);
+  const [searchType, setSearchFilter] = useState<SearchFilters>("TOKEN_ID");
+  const [selectedId, setSelectedId] = useState<undefined | string>(undefined);
```
Comment (pernielsentikaer) [inline extensions/tinyfaces-nft/src/tinyfaces.tsx:68]:
```suggestion
```
Leftover?

**Review by pernielsentikaer — COMMENTED** [review]
Hi 👋 

Thanks for your contribution 💪

Looks really good, only have a question about selectedId, is this something you forgot or just some leftover? 🙂

File: extensions/tinyfaces-nft/src/tinyfaces.tsx [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+            setSearchFilter(searchFilter);
+          }}
+        />
+      }
+    >
+      {tokenIdData && Boolean(searchText) && (
+        <List.Item
+          title={tokenIdData.name}
+          accessories={[{ text: `#${tokenIdData.token_id}` }]}
+          icon={{ source: `${tokenIdData.direct_url}?width=120&height=120`, mask: Image.Mask.Circle }}
```
Comment (pernielsentikaer) [inline extensions/tinyfaces-nft/src/tinyfaces.tsx:138]: The last thing I can think of if to have a [local fallback image](https://developers.raycast.com/api-reference/user-interface/icons-and-images#image.fallback) if images somehow don't load - could you add that and I think we're ready 🙂

I'm looking forward to testing this extension again 🔥

Comment (peduarte) [reply extensions/tinyfaces-nft/src/tinyfaces.tsx]: sounds good, done!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**